packages/angular/tsconfig.json

```diff
@@ -3,25 +3,27 @@
   "compilerOptions": {
     "baseUrl": "./",
     "outDir": "./dist/out-tsc",
-    "noImplicitAny": false,
-    "noImplicitReturns": false,
-    "noImplicitThis": false,
+    "forceConsistentCasingInFileNames": true,
```

Comment:
Changes here are to update `tsconfig.json` to similar as generated by @angular/cli@13